docker-compose.example.sqlite.yml

```diff
@@ -1,20 +1,26 @@
-version: "3.7"
+version: "2.4"
 services:
   app:
     image: babybuddy/babybuddy
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddy
     # See README.md#configuration for other environment configuration options.
     environment:
-      - ALLOWED_HOSTS=
+      - ALLOWED_HOSTS=localhost   # comma separated list of IP addresses or hosts that can access the web UI
       - DJANGO_SETTINGS_MODULE=babybuddy.settings.base
-      - SECRET_KEY=
-      - TIME_ZONE=
-      - DEBUG=False # Turn to False in production
+      - SECRET_KEY=               # Generate a random string here to secure the Django instance
+      - TIME_ZONE=                # In the tzdata format, IE, "America/Denver"
+      - DEBUG=False               # Turn to False in production
     volumes:
       - data:/app/data:rw
       - media:/app/media:rw
     command: bash -c 'python manage.py migrate --noinput && python manage.py createcachetable && gunicorn babybuddy.wsgi -b :8000 --log-level=info'
     ports:
       - "8000:8000"
+    # For Portainer deployment, select an alternate port for babybuddy, such as port 7000
```

Comment:
This would also require commenting out the port above, right? I think I'd rather see this information in the README instead of as a comment here. It could be a small additional section under the [Docker section](https://github.com/babybuddy/babybuddy/blob/master/README.md#docker).

Reply:
See updated alternate - instead of a commented option, I just reference the note in an inline comment. Does that look OK or should we leave it in the docs only?